### ISSUE TITLE
The use of USFederalHolidayCalendar will result in incorrect behavior

### ISSUE THREAD

**Issue by clearf** [pandas-dev/pandas#10278]
The definition of the USFederalHolidayCalendar is not correct. 

```
from pandas.tseries.holiday import USFederalHolidayCalendar
print(USFederalHolidayCalendar.rules)
```

> [Holiday: New Years Day (month=1, day=1, observance=<function nearest_workday at 0x10d17dcf8>), Holiday: Dr. Martin Luther King Jr. (month=1, day=1, offset=<DateOffset: kwds={'weekday': MO(+3)}>), Holiday: Presidents Day (month=2, day=1, offset=<DateOffset: kwds={'weekday': MO(+3)}>), Holiday: MemorialDay (month=5, day=24, offset=<DateOffset: kwds={'weekday': MO(+1)}>), Holiday: July 4th (month=7, day=4, observance=<function nearest_workday at 0x10d17dcf8>), Holiday: Labor Day (month=9, day=1, offset=<DateOffset: kwds={'weekday': MO(+1)}>), Holiday: Columbus Day (month=10, day=1, offset=<DateOffset: kwds={'weekday': MO(+2)}>), Holiday: Veterans Day (month=11, day=11, observance=<function nearest_workday at 0x10d17dcf8>), Holiday: Thanksgiving (month=11, day=1, offset=<DateOffset: kwds={'weekday': TH(+4)}>), Holiday: Christmas (month=12, day=25, observance=<function nearest_workday at 0x10d17dcf8>)]

```
print(USFederalHolidayCalendar.start_date)
```

> 1970-01-01 00:00:00

But Martin Luther King Day was first observed in 1986, so the holiday is being applied inappropriately before then. I am going to work on a snippet for a correct US holiday calendar.

**Comment by rockg** [comment]
Should be as easy as passing in `start_date` to the Holiday definition.

**Comment by OmerJog** [comment]
This is still an issue on master

**Comment by jreback** [comment]
@OmerJog hence the open tag

you are welcome to submit a patch

**Comment by roadswitcher** [comment]
Should this be closed?  
Per the comments on #10282, it looks like @clearf fixed it when 8ccdf4821f2dcac2b2c0f210d07679b5178f0211 was merged

**Comment by mroeschke** [comment]
Yes this looks fixed. Thanks for identifying. Closing
